index.js

```diff
@@ -359,6 +363,83 @@
       [a, b, a],
       function(x, y) { return x; });
 
+  //# A :: (a -> b) -> a -> b
+  //.
+  //. The A combinator. Takes a function and a value and returns the result of
+  //. applying the function to the value. Equivalent to Haskell's `($)` function.
+  //.
+  //. ```javascript
+  //. > S.A(R.add(1), 1)
+  //. 2
+  //.
+  //. > R.map(S.flip(S.A, 3), [R.add(2), R.multiply(2)])
+  //. [5, 6]
+  //. ```
+  S.A =
+  def('A',
+      {},
+      [$.Function, a, b],
+      function(f, x) { return f(x); });
+
+  //# C :: (a -> b -> c) -> b -> a -> c
+  //.
+  //. The C combinator. Takes a curried binary function and two values and
+  //. returns the result of applying the function to the values in reverse.
+  //. Equivalent to Haskell's `flip` function.
+  //.
+  //. This function is very similar to `S.flip`, except that its first argument
+  //. must be curried. This also allows it to work with manually curried
+  //. functions.
+  //.
+  //. ```javascript
+  //. > R.gt(1, 2)
+  //. false
+  //.
+  //. > S.C(R.gt, 1, 2)
+  //. true
```

Comment:
I actually had that with `"world!Hello "` before I changed it because I wanted to show a more practical example. I ended up showing how you can "fix" `R.gt`'s argument order using this combinator. I'll change it back if you think it's confusing though.